soc/riscv/riscv-ite/common/chip_chipregs.h

```diff
@@ -44,6 +44,9 @@
 #define REG_BASE_ADDR				EC_REG_BASE_ADDR
 #endif
 
+/* Common definition */
+#define EC_FREQ			MHZ(8)
```

Comment:
can the purpose/meaning be explained better here?

Reply:
Done